ProcessLib/BoundaryCondition/BoundaryConditionCollection.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef PROCESSLIB_BOUNDARYCONDITIONCOLLECTION_H
+#define PROCESSLIB_BOUNDARYCONDITIONCOLLECTION_H
+
+#include "DirichletBoundaryCondition.h"
+#include "ProcessLib/ProcessVariable.h"
+
+namespace ProcessLib
+{
+
+class BoundaryConditionCollection : public BoundaryCondition
```

Comment:
I could remove the inheritance, since this class is not meant to be used polymorphically.
The inheritance is there to stress that a BC collection in the end is only a BC (a special one, which happens to be a collection of several individual BCs), so I'd like to keep it.